homeassistant/components/habitica/sensor.py

```diff
@@ -19,26 +21,34 @@
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
 
-ST = SensorType = namedtuple("SensorType", ["name", "icon", "unit", "path"])
+SensorType = namedtuple("SensorType", ["name", "icon", "unit", "path"])
```

Comment:
Mypy did not like at all the double-assignment here, and it seems `ST` was only used inside the `SENSORS_TYPES` and `TASKS_TYPES` definitions below.
Note to the code-owner => this would fit nicely in entity descriptions.